subsys/bluetooth/host/gatt.c

```diff
@@ -3606,7 +3607,15 @@ static int ccc_set(const char *name, size_t len_rd, settings_read_cb read_cb,
 	return 0;
 }
 
-BT_SETTINGS_DEFINE(ccc, ccc_set, NULL, NULL);
+const struct settings_handler_stat bt_ccc_handler = {
+	.name = "bt/ccc",
+	.h_get = NULL,
```

Comment:
Any struct members not explicitly initialised are implicitly initialised to NULL. At least in the Bluetooth code we don't generally list any members in the initialisation that we don't use. I don't have a super strong opinion on this though, but it'd be good to keep some consistency throughout the code base.

Reply:
As you wish, I added them (consistently) to the bluetooth module to allow developers to quickly see how the elements are called. Since in the bluetooth code they are generally omitted I will do the same.